packages/components/src/form-token-field/style.scss

```diff
@@ -46,10 +46,15 @@
 }
 
 .components-form-token-field__label {
-	display: inline-block;
+	display: block;
 	margin-bottom: $grid-size-small;
 }
 
+.components-form-token-field__help {
+	display: block;
+	font-style: italic;
```

Comment:
I'd like to have some design feedback about the italic style. Personally, I'd avoid it because some users find it difficult to read.

Reply:
I have made it italic based on styling of the help text of `BaseControl` which is used as a base for most of other controls. If there are italics missing for help texts similar to this, they are inconsistent with the rest. Just a few examples:

<img width="180" alt="Screenshot 2019-05-06 at 21 18 19" src="https://user-images.githubusercontent.com/156676/57249367-088ca480-7045-11e9-9a63-7341f4996cb0.png">
<img width="180" alt="Screenshot 2019-05-06 at 21 18 29" src="https://user-images.githubusercontent.com/156676/57249369-0a566800-7045-11e9-9298-355452b64a28.png">
<img width="180" alt="Screenshot 2019-05-06 at 21 19 10" src="https://user-images.githubusercontent.com/156676/57249370-0cb8c200-7045-11e9-8d37-56950d401efe.png">

I agree on the issue with italics but I think we should address that separately and on a global level for all those existing usages of BaseControl. What do you think about leaving this PR as italics and opening a discussion to replace them globally for all help texts? I'll be happy to make that PR once we settle on a solution.

Reply:
Yeah, I think italics vs. no italics can be discussed separately — this PR is just mirroring the existing styles used in `.components-base-control__help`. 

(From some quick clicking around, it seems that the [Permalink help text](https://github.com/WordPress/gutenberg/pull/15469#pullrequestreview-234087938) is the only text of this kind that isn't italicized, so we should make a decision one way or another).

Reply:
Italic is terrible for some kind of dyslexia when used on whole sentences. I vote for just not add new Italic. Also, planning to create related GitHub issue and Trac ticket. Thank you.

Reply:
Cool — I'm fine getting rid of the italic globally if we can replace it with something else. I just want to make sure that help text appears differently from the form field label so users can tell them apart. But there are other ways to do that besides making the text italic. I'll keep an eye out for the ticket and try out some ideas there. 🙂

Reply:
> there are other ways to do that besides making the text italic
Definitely agree. One day before I die I'd like to see a typographic system in WordPress 🙂

Reply:
I've opened https://github.com/WordPress/gutenberg/issues/15683 to track the inconsistent help text, and to consider the use of italic text there. We'll get that sorted out globally in Gutenberg. 🙂

In the meantime, that shouldn't block this PR since it's not adding any new styles — just replicating the existing `BaseControl` help text style.